.circleci/config.yml

```diff
@@ -348,4 +348,4 @@ workflows:
             - python312
             - py39cassandra
             - py39couchbase
-            - py39gevent
+            - py39gevent
```

Comment:
Why the change here? In general we prefer newlines at the end of the file.

Reply:
I think I removed it by mistake. Fixed this in the latest commit.